content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md

```diff
@@ -0,0 +1,146 @@
+6. Note that the gas fee for the cancelation is 0.00001 ETC.
+
+![Canceling a sent transaction.](./etc-metamask-cancel.png)
+
+## 6. Speed Up Sent Transactions
+
+Similarly as cancelling a transaction, on ETC you can also speed up transactions that have not yet been included in a block. To do this follow these instructions:
+
+1. Send a transaction as explained above (in this example we are sending 0.99993).
+2. In the next screen you will see a confirmation of your transaction submission, but if you wish to speed it up, press the balance area.
+3. In the next step you will see the submitted transaction with two buttons below, “Speed up” and “Cancel”, press the “Speed up” button.
+4. In the next screen press “Yes, let’s try”.
+5. The speed up transaction will be sent and your transaction should get executed faster.
+6. Note that the gas fee for the speedup is 0.00001 ETC.
+
+![Speeding up a transaction.](./etc-metamask-speed-up.png)
+
+## 7. Create More ETC Addresses
+
+For you convenience, to categorize your crypto assets, or for privacy reasons, you can have many ETC addresses inside Metamask. To add new addresses follow these instructions:
+
+1. Make sure you are on the ETC network, then tap on the icon above the account you are on.
+2. In the next screen, MetaMask will have several options. Select “Create a new account”.
+3. Immediately, a new account will be generated in the screen and will be assigned the next number in the account list. In this example it is “Account 2”.
+4. If you want to use that account, tap it to select it.
+5. When you go back to the home, you will see that account as the default account. 
+
+![Creating more ETC addresses.](./etc-metamask-create-address.png)
+
+## 8. Use the Same Addresses in Ethereum Classic or in Ethereum
+
+Because ETC and ETH are totally compatible, then you can actually use the same addresses interchangeably. This is, you can use your same ETC address inside the ETH blockchain and vice versa.
+
+To do this, just go to the address you want to use, then change the network on the network dropdown menu and select which one you wish to use.
+
+The address will work on both chains.
+
+![Using the same address on ETC and ETH.](./etc-metamask-etc-eth.png)
+
+## 9. Back Up Your Wallet
+
+It is imperative that you back up your MetaMask wallet because if you lose your phone or your data you will lose access to all your crypto.
+
+To do this, MetaMask has a 12 word secret passphrase that is like a password that may be used to regenerate all your addresses. Once you back it up, you can import your 12 words again in a new device and recover your assets.
+
+Your backup should be in paper and not in digital form because if you store your words in a computer or phone, they may be lost, compromised, or hacked.
+
+![Back up your 12 word secret passphrase and MetaMask password on paper.](./etc-metamask-back-up.png)
+
+Once you have it in paper, you should make a couple of copies and keep them in safe places.
```

Comment:
```suggestion
Once you have it on paper, you should make a couple of copies and keep them in safe places.
```

Reply:
I will add your typo and address disclaimer suggestions after publishing. The article approval process is uncertain and I prioritize publishing on time over small typos and the disclaimer is about clarifying that using ETC and ETH addresses interchangeably in general may cause losses, but this post is specifically about MetaMask which means the user does have the private keys of the addresses whether they use them on ETH or ETC.